wrappers/python/indy/error.py

```diff
@@ -92,6 +92,9 @@ class ErrorCode(IntEnum):
     # Attempt to send transaction without the necessary privileges
     LedgerSecurityError = 305,
 
+    # Attempt to create pool with name used for another exists pool
+    PoolLedgerAlreadyExistsError = 306,
```

Comment:
May be it is better to rename this to PoolLedgerConfigAlreadyExistsError. Let's discuss offline.